docs/data_munging.md

```diff
@@ -0,0 +1,76 @@
+# Technical documentation for We Predicted That pokemon battle predictor
+
+## Step 1- Data Munging
+
+The first step of any machine learning project is "data munging", i.e. ensuring that your dataset is suitable for feeding into your model. This includes filling in gaps in the data (or removing columns if they are too incomplete), sorting the data, filtering, standardising, etc.
+
+Fortunately for us, the Pokemon dataset we used was mostly complete - only one Pokemon name was missing so this could be filled in manually. However, the data was broken into multiple tables: `combats`, `pokemon_data` and `tests`. The `combats` table contained two pokemon ids, and the winner:
+
+```
+First_pokemon,Second_pokemon,Winner
+266,298,298
+702,701,701
+191,668,668
+...
+````
+
+As mentioned above, predicted a winner in a fight between two pokemon is a binary classification problem, since there are only two outcomes. In this dataset, the winner is identified by their id so there are 800 outcomes! The first step was therefore to identify the winner relative to the pokemon fighting so we have either pokemon 1 (0) or pokemon 2 (0) as the winner. The data munging was done in the terminal, rather than saved as a script, since it was a one-off process:
```

Comment:
predicted/predicting - first sentence

Is it right to have "pokemon 1 (0) or pokemon 2 (0)"?